pkg/apis/core/validation/validation_test.go

```diff
@@ -6749,10 +6749,13 @@ func TestValidateResizePolicy(t *testing.T) {
 	tSupportedResizeResources := sets.NewString(string(core.ResourceCPU), string(core.ResourceMemory))
 	tSupportedResizePolicies := sets.NewString(string(core.NotRequired), string(core.RestartContainer))
 	type T struct {
-		PolicyList  []core.ContainerResizePolicy
-		ExpectError bool
-		Errors      field.ErrorList
+		PolicyList       []core.ContainerResizePolicy
+		ExpectError      bool
+		Errors           field.ErrorList
+		PodRestartPolicy core.RestartPolicy
 	}
+	// var PodRestartPolicy core.RestartPolicy
+	// PodRestartPolicy = "Never"
 	testCases := map[string]T{
 		"ValidCPUandMemoryPolicies": {
 			[]core.ContainerResizePolicy{
```

Comment:
It would be really nice to change this to use named fields in all cases - it's hard to read without.

If you do this, please do it as a separate commit BEFORE this one

Reply:
You can do it as a new PR or just as a commit on this PR before this commit